templates/c2s-lb/0/rancher-compose.yml

```diff
@@ -50,6 +56,13 @@ services:
         service: ${profiles_frontend_service}
         source_port: 443
         target_port: 3000
+      - hostname: ''
+        path: /gp-finders
```

Comment:
I'd go with `/gp-finder` as I see it as a singular rather than the plural of both `/profiles` and '/finders`. What do you reckon?